.eslintrc.cjs

```diff
@@ -9,5 +9,13 @@ module.exports = {
   rules: {
     'no-shadow': 'off',
     '@typescript-eslint/no-shadow': ['error'],
+    '@typescript-eslint/no-unused-vars': [
```

Comment:
Added this so that parameters that are prefixed with an underscore are exempt from the rule. This is a trick I picked up from @stephen-carter-at-sf .